test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -805,6 +805,17 @@ describe('Dropdown Component', () => {
       newItem.should.have.prop('text', 'bar')
       newItem.should.have.prop('value', 'bar')
     })
+
+    it('passes options as props', () => {
+      const customOptions = [
+        { text: 'abra', value: 'abra', className: 'customClass' },
+        { text: 'cadabra', value: 'cadabra', className: 'customClass' },
+        { text: 'bang', value: 'bang', className: 'customClass' },
```

Comment:
Since className is a likely handled prop, how about adding a `data-foo` prop in here to be completely sure unanticipated props are passed?